package/com.unity.formats.usd/Tests/Editor/EditorTests.cs

```diff
@@ -67,6 +75,29 @@ public void LoadAsFbxCompareMeshTransforms()
             Assert.AreApproximatelyEqual(usdMeshTr.localRotation.y,fbxMeshTr.localRotation.y);
             Assert.AreApproximatelyEqual(usdMeshTr.localRotation.z,fbxMeshTr.localRotation.z);
         }
+
+        [Test]
+        public void TestLeftHandedUsdImport()
```

Comment:
there is probably a better way to test this, adding this for now to check that the same cube is handled differently whether it is right or left handed.